index.rst

```diff
@@ -2,24 +2,54 @@
 
 .. sectnum::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+Abstract
+========
 
-.. TODO: Delete the note below before merging new content to the main branch.
+This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
 
-.. note::
+Description of the Incident
+===========================
 
-   **This technote is a work-in-progress.**
+This incident initiated between 8:00-8:30 CLT/3:00-3:30PST December, 6 2022, during the first night of the 2022A run on the Auxiliary Telescope.
+We experienced a catastrophic network failure that took down the network in the entire observatory.
+The failure extended to both the internal and external network, both over the WiFi and the cable connections.
+All WiFi connections were dropped and we could no longer connect to the network.
 
-Abstract
-========
+Up until the issue started we were preparing to go on sky with the Auxiliary Telescope.
+The TMA was experiencing power issues and would not be able to operate during the night for the soak and start tracker testing. 
+Alysha Shugart and Tiago Ribeiro were executing some dome movement tests with the remote help of Bruno Quint.
 
-This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
+At one point around 8:30 PM local time, Tiago Ribeiro went to update LOVE in the love02 node with the latest version of the logging system for testing during the night.
+The node was completely shutdown (no LOVE system or ospl daemon running), shortly after the ospl daemon on love2 was initialized the network outage started.
```

Comment:
nitpick: is it love2 or love02?  the document has both
another nitpick: a little confused with "no LOVE system or ospl daemon running" - as they were both running on azar1 - correct?  maybe just add "on the node" for clarification
Maybe go in depth here on what we have set up for the LOVE system - to better explain why the accident occurred - as it might not be very clear for the uninitiated. 
Also don't we have a confluence page that lists all the IP addresses?  Should we mention that here?
Should we go more in-depth on the process of initializing the LOVE/ospl daemon to show there aren't any other mitigations that can take place?

Reply:
I replaced all love2 with love02.

I expanded a but the details so I think it address some of your doubts.. I am not sure how much detail would be useful here. I am not sure going more in-depth on the process would be helpful, but I could maybe point to the documentation?